Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Private/QTMConnectLiveLinkSource.cpp

```diff
@@ -36,13 +36,37 @@ QTMConnectLiveLinkSettings QTMConnectLiveLinkSettings::FromString(const FString&
         if (autoDiscover == "true")
             settings.AutoDiscover = true;
     }
+    settings.Stream3d = true;
+    FString stream3d;
+    if (!FParse::Value(*settingsString, TEXT("Stream3d="), stream3d))
+    {
+        if (stream3d == "false")
+            settings.Stream3d = false;
+    }
+    settings.Stream6d = true;
+    FString stream6d;
+    if (!FParse::Value(*settingsString, TEXT("Stream6d="), stream6d))
+    {
+        if (stream6d == "false")
+            settings.Stream6d = false;
+    }
+    settings.StreamSkeleton = true;
+    FString streamSkeleton;
+    if (!FParse::Value(*settingsString, TEXT("StreamSkeleton="), streamSkeleton))
+    {
+        if (streamSkeleton == "false")
+            settings.StreamSkeleton = false;
+    }
     return settings;
 }
 
 FString QTMConnectLiveLinkSettings::ToString() const
 {
     FString settingsString = FString::Printf(TEXT("IpAddress=\"%s\""), *IpAddress);
     settingsString.Append(FString::Printf(TEXT("AutoDiscover=\"%d\""), AutoDiscover ? TEXT("true") : TEXT("false")));
+    settingsString.Append(FString::Printf(TEXT("Stream3d=\"%d\""), Stream3d ? TEXT("true") : TEXT("false")));
+    settingsString.Append(FString::Printf(TEXT("Stream6d=\"%d\""), Stream6d ? TEXT("true") : TEXT("false")));
```

Comment:
Change GUI labels to `Stream Markers` and `Stream Rigid Bodies`.

Reply:
That is not a gui label. The settings are sent as a string between the livelink editor and livelink plugin. Don't need to change internal naming.